web/src/components/upload/confirm.js

```diff
@@ -157,7 +157,7 @@ export const UploadConfirm = ({
               disabled
               name="device"
               id="device"
-              value={dustrakSerialToLabel[dustrakSerial]}
+              value={dustrakSerialToLabel}
```

Comment:
In line with changing this to be the whole device object, we'll want to change the property that we pass. We'll also want to change the prop types and JSDoc that explain what properties we're passing. Finally, we'll want to change the the way we use the property